tsconfig.json

```diff
@@ -1,46 +1,52 @@
 {
   "compilerOptions": {
-    "lib": [
-      "esnext",
-      "dom",
-      "dom.iterable",
-      "scripthost"
-    ],
+    /* Language and Environment */
     "target": "esnext",
-    "outDir": "lib",
-    "declaration": true,
+    "lib": [ "esnext", "dom", "dom.iterable", "scripthost"],
+    "jsx": "preserve",
+    // todo: 待移除 哪里有使用装饰器吗
+    "experimentalDecorators": true,
+
+    /* Modules */
     "module": "esnext",
+    "rootDir": ".",
+    // todo: 待更新 按照官网介绍，node 和 node10 已经弃用了，https://www.typescriptlang.org/docs/handbook/modules/theory.html#module-resolution-is-host-defined
+    // 但如果用 node16 的话， module 需要使用 node，因此这里我们讨论一下吧
```

Comment:
bundler is better https://github.com/microsoft/TypeScript/pull/51669 after upgrade to v5